Forward and reverse reads from two lanes were merged per sample. Quality control of raw and trimmed reads was conducted using FastQC (v0.11.9), and adapter trimming was performed with fastp (v0.22.0) using default parameters. Reads were aligned to the T2T-CHM13v2.0 reference genome (GCF_009914755.1) using HISAT2 (v2.2.1). SAM files were converted and sorted to BAM format using Samtools (v1.16.1). Gene-level counts were generated using FeatureCounts (Subread v2.0.7) with the following options: exon-level counting (-t exon), gene-level summarization (-g gene_id), paired-end mode (-p), and reverse-strand specificity (-s 2), using the corresponding GTF annotation file.
